src/components/PauseSubscriptionPopup/PauseSubscriptionPopup.js

```diff
@@ -0,0 +1,280 @@
+    );
+  }
+
+  const pausePeriod = periodMapper[toOffer.period].chargedForEveryText;
+  const currencySymbol = currencyFormat[fromOffer.nextPaymentCurrency];
+  const currentPrice = formatNumber(fromOffer.nextPaymentPrice);
+  const pauseStartingDate = dateFormat(fromOffer.expiresAt);
+
+  return (
+    <InnerPopupWrapper
+      steps={isPartOfCancellationFlow ? 3 : 2}
+      popupTitle={t('Pausing plan')}
+      currentStep={
+        isPartOfCancellationFlow ? STEPS_NUMBERS[step] + 1 : STEPS_NUMBERS[step]
+      }
+    >
+      {step === STEPS.PAUSE_DETAILS && (
+        <>
+          <ContentStyled>
+            <ImageWrapper>
+              <SubscriptionIconStyled
+                period={fromOffer.period}
+                showLabel="Current"
+                gray
+              />
+              <ArrowStyled />
+              <SubscriptionIconStyled isPaused showLabel="Paused" />
+            </ImageWrapper>
+            <TitleStyled step={step} textTransform="capitalize">
+              {t('Subscription pause')}
+            </TitleStyled>
+            <TextStyled>
+              <Trans i18nKey="pausesubscriptionpopup-info">
+                Your subscription will be paused for {{ pausePeriod }} starting{' '}
+                <strong>{{ pauseStartingDate }}</strong>. During the
+                subscription pause period, you will not be charged. Your
+                subscription will be automatically resumed after a{' '}
+                {{ pausePeriod }} and you will continue to be charged{' '}
+                <strong>
+                  {{ currencySymbol }}
+                  {{ currentPrice }}
+                </strong>{' '}
+                on a recurring basis.
+              </Trans>
+            </TextStyled>
+            <TextStyled>
+              {t('You can resume your subscription at any time.')}
+            </TextStyled>
+            <TextStyled step={step}>
+              {t('Do you want to apply the change now?')}
+            </TextStyled>
+          </ContentStyled>
+          <ButtonWrapperStyled removeMargin>
+            <Button
+              theme="simple"
+              onClickFn={() => {
+                if (isPartOfCancellationFlow) {
+                  showInnerPopup({
+                    type: POPUP_TYPES.updateSubscription,
+                    data: {
+                      action: 'unsubscribe',
+                      offerData: {
+                        ...fromOffer
+                      }
+                    }
+                  });
+                } else {
+                  eventDispatcher(MSSDK_SWITCH_POPUP_ACTION_CANCELLED, {
+                    fromOfferId: fromOffer.offerId,
+                    toOfferId: toOffer.toOfferId,
+                    switchDirection: toOffer.switchDirection,
+                    algorithm: toOffer.algorithm
+                  });
+                  hideInnerPopup();
+                }
+              }}
+            >
+              {t('Continue current Plan')}
```

Comment:
onClickFn in case of isPartOfCancellationFlow moves to unsubscribe action, but the button says 'Continue current Plan'. I think we should adjust the copy here